### PR TITLE
Remove no longer used variables.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: charts/mozmoderator/values.yaml [hunk 1]
```diff
@@ -13,8 +13,6 @@ configMap:
     ANON_ALWAYS: true
     AWS_DEFAULT_REGION: us-west-2
     FROM_NOREPLY: Mozilla Moderator <no-reply@localhost>
-    MOZILLIANS_API_BASE: https://mozillians.org/api/v1/users/
```
Comment (akatsoulas) [inline charts/mozmoderator/values.yaml:16]: These variables are no longer used by Moderator. Moreover mozillians.org is decommissioned.

Comment (cmharlow) [reply charts/mozmoderator/values.yaml:16]: great, thanks for this!

Comment (cmharlow) [reply charts/mozmoderator/values.yaml:16]: going to get this in with failing CI then make another PR updating a few spots & tag you on it Tasos, so you can see a few places these variables get picked up.

Comment (cmharlow) [reply charts/mozmoderator/values.yaml:16]: (i don't see a good way to propose changes on lines not yet touched but affected by these variables being removed)

**Review by cmharlow — APPROVED** [review]
Demoing changes for charts in a future PR, aware CI is failing.

**Pull request merged**